src/app/clients/add/ClientForm.tsx

```diff
@@ -34,15 +33,17 @@ import DeliveryInstructionsCard from "@/app/clients/add/formSections/DeliveryIns
 import ExtraInformationCard from "@/app/clients/add/formSections/ExtraInformationCard";
 import AttentionFlagCard from "@/app/clients/add/formSections/AttentionFlagCard";
 import SignpostingCallCard from "@/app/clients/add/formSections/SignpostingCallCard";
-import {
-    ClientDatabaseInsertRecord,
-    deleteFailedInsert,
-    insertClient,
-    insertFamily,
-} from "@/app/clients/add/databaseFunctions";
 import Button from "@mui/material/Button";
+import { submitAddClientForm, submitEditClientForm } from "@/app/clients/add/submitFormHelpers";
+
+interface Props {
+    initialFields: Fields;
+    initialFormErrors: FormErrors;
+    editMode: boolean;
+    clientID?: string;
+}
 
-interface AddClientFields extends Fields {
+export interface ClientFields extends Fields {
```

Comment:
Actually what is the purpose of `extends Fields` - are there any field entries that are not listed in this interface?

Reply:
there could be - it's so that it's more extendable (?? - feel like that's the wrong word) in the future

Reply:
The problem is though like the `Fields` interface allows the user to put in props that are not in e.g. ClientFields and they will pass the type check because `Fields` is so general that it will accept all the props that you have put in
I am just wondering because whenever you make a form, the developer should be clear about what are the props gonna be fed in (in this case, all the necessary props are defined in interface `ClientFields` already so I am not sure when will the `Fields` interface come into play
P.S I think the parcel form didn't extend if I didn't remember wrongly? And things are working